.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -30,11 +24,19 @@ is closed automatically when this PR is merged.
 Checklist
 ---------
 
-- [ ] I have read the [Contribution Guidelines](https://github.com/restic/restic/blob/master/CONTRIBUTING.md#providing-patches)
-- [ ] I have enabled [maintainer edits for this PR](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)
-- [ ] I have added tests for all changes in this PR
-- [ ] I have added documentation for the changes (in the manual)
-- [ ] There's a new file in `changelog/unreleased/` that describes the changes for our users (template [here](https://github.com/restic/restic/blob/master/changelog/TEMPLATE))
-- [ ] I have run `gofmt` on the code in all commits
-- [ ] All commit messages are formatted in the same style as [the other commits in the repo](https://github.com/restic/restic/blob/master/CONTRIBUTING.md#git-commits)
-- [ ] I'm done, this Pull Request is ready for review
+<!--
+Fill out this checklist by replacing [ ] with [x].
+
```

Comment:
Please remove these two lines and instead add "Enable a checkbox by replacing [ ] with [x]" (or a better wording if you have one) at the end of the paragraph below. This makes it more concise.

Reply:
I've updated the template.